battleship.py

```diff
@@ -80,15 +92,22 @@ def createShip():
 Returns: bool
 '''
 def checkShip(grid, ship):
-    return
+    count=0
+    for i in ship:
+        if grid[i[0]][i[1]] == EMPTY_UNCLICKED:
+            count+=1
+            if count==3:
+               return True
+        else:
+            return False

```

Comment:
You can reduce the number of lines in the function by checking the not equal condition and returning False, and returning True on completion of the loop.

Reply:
@Krishnasaianumandla please make the requested changes.

Reply:
Resolved the problems.